deployments/embed.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package embed generates plugin DaemonSet object out of plugin yaml.
+package embed
+
+import (
+	_ "embed"
+
+	apps "k8s.io/api/apps/v1"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	//go:embed dsa_plugin/base/*plugin.yaml
+	contentDSA         []byte
+	DSAPluginDaemonSet = getDaemonset(contentDSA)
+
+	//go:embed fpga_plugin/base/*plugin*.yaml
+	contentFPGA         []byte
+	FPGAPluginDaemonSet = getDaemonset((contentFPGA))
+
+	//go:embed gpu_plugin/base/*plugin.yaml
+	contentGPU         []byte
+	GPUPluginDaemonSet = getDaemonset(contentGPU)
+
+	//go:embed qat_plugin/base/*qat-plugin.yaml
+	contentQAT         []byte
+	QATPluginDaemonSet = getDaemonset(contentQAT)
+
+	//go:embed sgx_plugin/base/*plugin*.yaml
+	contentSGX         []byte
+	SGXPluginDaemonSet = getDaemonset(contentSGX)
+
+	//go:embed vpu_plugin/base/*plugin*.yaml
+	contentVPU         []byte
+	VPUPluginDaemonSet = getDaemonset(contentVPU)
```

Comment:
VPU can be dropped because we have no controller for it.

Reply:
done